bats/fb-content-katello.bats

```diff
@@ -10,91 +10,97 @@ setup() {
   tSetOSVersion
 }
 
+@test "Create an Empty Organization" {
+  run hammer -u admin -p changeme organization info --name "Empty Organization"
```

Comment:
@ekohl this was the  magic I was looking for so that this didn't cause a test failure